setup.cfg

```diff
@@ -61,6 +62,14 @@ commands =
     --no-license-path \
     --output-file=./licenses/license.txt
 
+# tox test environment for testing ray optional dependency imports
```

Comment:
update comment (environment for testing imports without any optional dependencies installed) - careful to distinguish the descriptions of `default` (no deps) and `all` (all deps).

Reply:
Good idea!